src/states/menus/lobby.gd

```diff
@@ -236,6 +236,7 @@ func generate_button_grid() -> void:
 		var button := preload("res://src/states/menus/buttons/color_button.tscn").instantiate() as ColorButton
 		button.set_button_color(COLORS[angle_idx])
 		button_circle.add_child(button)
+		button.owner = self
```

Comment:
What are these here in the lobby/scoreboard for. It doesn't look like they have an associated multiplayerspawner?